docs/assemblies/assembly-aws-kms.adoc

```diff
@@ -0,0 +1,14 @@
+// file included in the following:
```

Comment:
KMS currently belongs to exclusively to the record-encryption filter, so to see it further up the tree looks surprising.

Reply:
I agree that KMSs will probably end up as a concept in Kroxylicious that is used by n filters.  On reflection, I think the path structure suggest by Paul is okay.